iOSDFULibrary/Classes/Utilities/Streams/DFUStreamZip.swift

```diff
@@ -174,7 +175,7 @@ internal class DFUStreamZip : DFUStream {
                 }
                 
                 let applicationType = FIRMWARE_TYPE_APPLICATION
-                if type & applicationType == applicationType {
+                if type.rawValue & applicationType == applicationType {
```

Comment:
Instead of using raw value can't we set `FIRMWARE_TYPE_APPLICATION` and to the equivalent enum of DFUFirmwareType ?
this is not a breaker but it would be a tiny bit cleaner

Reply:
Hmm.. now I understand. Will check.

Reply:
We can replace FIRMWARE_TYPE.. by DFUFirmwareType.xxx.rawValue, but can't by enum values. I'm comparing bit fields.

Reply:
Enums can do the same, let's leave it as is for now and have a new update that will do some minor cleanups here and there 👍

Reply:
For future reference, this looks like a cleaner candidate for us:
https://developer.apple.com/documentation/swift/optionset

Reply:
Maybe next release. This may break backwards compatibility.